### ISSUE TITLE
Pause button is not announced in VoiceOver

### ISSUE THREAD

**Issue by sbates-idrc** [codelearncreate/c2lc-coding-environment#478]
The play button becomes the pause button visually when the program is running.  With VoiceOver, the button is initially announced as the play button, and no announcement is made to indicate the the user is now focused on the pause button.  I would suggest simply announcing it as the "play/pause button" or something similar.

The change from play to pause is announced in NVDA.

This was an issue that came out of a session with a screen reader user.

Original Jira issue: https://fluidproject.atlassian.net/browse/C2LC-571

**Comment by sbates-idrc** [comment]
We discussed the Play/Pause button at the IDRC tech meeting on May 2, 2024.

- Ned mentioned https://inclusive-components.design/toggle-button/
- Joseph suggested:
  - https://www.w3.org/WAI/ARIA/apg/patterns/button/#examples
  - https://w3c.github.io/core-aam/#mapping_events_state-change
    - On Mac, a change in `aria-label` is mapped to `AXLabelCreated` (note “created” rather than “changed”; if macOS is unable to express a label change, then it is not surprising that VoiceOver does not announce a label change)
  - https://developer.apple.com/documentation/appkit/nsaccessibility
